spec/services/zendesk_cleaner_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe ZendeskCleaner do
+  let(:client) { double(ZendeskAPI::Client) }
+  let(:ticket1) { double(ZendeskAPI::Ticket, id: 1) }
+  let(:ticket2) { double(ZendeskAPI::Ticket, id: 2) }
+  let(:ticket3) { double(ZendeskAPI::Ticket, id: 3) }
+  let(:tickets) { double(ZendeskAPI::Collection) }
+  let(:ticket_ids) { [ticket1.id, ticket2.id, ticket3.id] }
+
+  subject { described_class.new }
+
+  context 'when Zendesk is configured' do
+    it 'successfully bulk deletes tickets older than 12 months old' do
+      Rails.configuration.zendesk_client = client
```

Comment:
you want to use `set_configuration_with(:zendesk_client, client)` because here you're mutating the zendesk client for all the tests